wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -917,6 +922,12 @@ private function execute_create_table() {
 				$constraint->value
 			);
 		}
+
+		foreach ( $table->fields as $field ) {
+			if ( $field->on_update ) {
+				$this->add_column_on_update_current_timestamp( $table->name, $field->name );
+			}
+		}
```

Comment:
@JanJakes from what I can see we don't check for `CURRENT_TIMESTAMP` anywhere.
I tried replacing `CURRENT_TIMESTAMP` in tests with `NOT_CURRENT_TIMESTAMP` and they still passed.

Could we run `$this->add_column_on_update_current_timestamp` only if the query includes `ON UPDATE CURRENT_TIMESTAMP`?

Reply:
@bgrgicak 👍 Good point. There is no other `ON UPDATE SOMETHING` construct for columns in MySQL, but it's better to be more explicit. Updated in https://github.com/WordPress/sqlite-database-integration/pull/150/commits/ea37a9f6ded2ea77fc080df5f72be30bea1416e3.